packages/sampler/stories/qa/Picker.js

```diff
@@ -1,12 +1,12 @@
 import Picker from '@enact/moonstone/Picker';
-import {icons} from '@enact/moonstone/Icon';
 import PickerAddRemove from './components/PickerAddRemove';
 import PickerRTL from './components/PickerRTL';
 import React from 'react';
 import {storiesOf} from '@storybook/react';
 
-import {boolean, select} from '../../src/enact-knobs';
 import {action} from '../../src/utils';
+import {boolean, select} from '../../src/enact-knobs';
+import iconNames from '../default/icons';
```

Comment:
In the future, we might consider moving the icons file into a shared folder rather than referencing `default` from `qa`.